misc/userscripts/qute-pass

```diff
@@ -160,11 +161,25 @@ def find_pass_candidates(domain, unfiltered=False):
 
 
 def _run_pass(pass_arguments):
-    # The executable is conveniently named after it's mode [pass|gopass].
-    pass_command = [arguments.mode]
+    # The executable is conveniently named after its mode [pass|gopass].
+    pass_command = [arguments.mode] + pass_arguments
     env = os.environ.copy()
     env['PASSWORD_STORE_DIR'] = arguments.password_store
-    process = subprocess.run(pass_command + pass_arguments, env=env, stdout=subprocess.PIPE)
+    process = subprocess.run(pass_command, env=env, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    err_msg = process.stderr.decode(arguments.io_encoding).strip()
+
+    if err_msg:
+        if arguments.mode == "gopass" and "initialize" in err_msg:
+            # Warn the user if the error could be caused by a changed default directory.
+            # See <https://github.com/gopasspw/gopass/pull/1968/commits/de4ba223422417559f36346bd51ce075e9812924>
+            stderr(
+                "Warning: With gopass 1.12.8, the default password store directory changed from `~/.password-store` " \
+                "to `~/.local/share/gopass/stores/root, and qute-pass now uses this new default. If your password " \
+                "store isn't '{}', specify a store with the `--password-store` argument to qute-pass, or by setting " \
+                "the PASSWORD_STORE_DIR environment variable.".format(arguments.password_store)
```

Comment:
```suggestion
                "Warning: With gopass 1.12.8, the default password store directory changed from `~/.password-store` "
                "to `~/.local/share/gopass/stores/root, and qute-pass now uses this new default. If your password "
                f"store isn't '{arguments.password_store}', specify a store with the `--password-store` argument to qute-pass, or by setting "
                "the PASSWORD_STORE_DIR environment variable."
```

using more modern f-strings and removing unneeded backslashes (newline inside `(...)` are ignored by Python).
Might need to readjust the line breaks though.

Reply:
Agreed! I initially tried to keep changes minimal, but as Python 3.7 is a requirement anyway, I took the opportunity now to clear out all old `.format()`s and replace them with f-strings (and break up some long lines).